docs/docs/protocol-specs/gas-and-fees/published-gas-and-fee-data.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Published Gas & Fee Data
+---
+
+# Published Gas & Fee Data
+
+When a block is published to L1, it includes information about the gas and fees at a block-level, and at a transaction-level.
+
+## Block-level Data
+
+The block header contains a `GlobalVariables`, which contains a `GasFees` object. This object contains the following fields:
```

Comment:
We will want to publish the total amount of gas used in the block I suspect.

Reply:
I'm not sure. If you're saying that because we will want to meter against a block target, and subsequently update these `feePer` values, we could instead compute the sum during the rollup circuits and publish the block with the new `feePer` values (which the following block would use for its fees). 

I'm not sure why else we would publish them.